libc-test/build.rs

```diff
@@ -3918,6 +3920,10 @@ fn test_linux(target: &str) {
         if sparc64 && (ty == "Elf32_Rela" || ty == "Elf64_Rela") {
             return true;
         }
+        // FIXME: alignment issue with this arch
+        if loongarch64 && ty == "ethhdr" {
+            return true;
+        }
```

Comment:
Any idea why? There doesn't seem to be anything loongarch-specific at https://github.com/torvalds/linux/blob/e53642b87a4f4b03a8d7e5f8507fc3cd0c595ea6/include/uapi/linux/if_ether.h#L172-L178. If it was just a bug then maybe it's worth removing to retry.